src/schema/v2/artwork/index.ts

```diff
@@ -1542,18 +1571,27 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         type: Money,
         description:
           "The price paid for the artwork in a user's 'my collection'",
-        resolve: (artwork) => {
-          const { price_paid_cents } = artwork
+        resolve: async (artwork, args, ctx, info) => {
+          const { price_paid_cents, price_paid_currency = "USD" } = artwork
```

Comment:
Slightly surprised that we ask collectors to enter price in cents, but I guess it was here already.

Reply:
It is probably calculated on the artwork from some input using the money helper gem